yt_dlp/extractor/moviepilot.py

```diff
@@ -89,31 +90,27 @@ class MoviepilotIE(InfoExtractor):
         },
     }]
 
-    def _get_property(self, name, webpage, fatal=True):
-        return self._search_regex(f'meta itemprop="{name}" content="(.*?)"', webpage, name, fatal=fatal)
-
     def _real_extract(self, url):
         video_id = self._match_id(url)
 
         webpage = self._download_webpage(f'https://www.moviepilot.de/movies/{video_id}/trailer', video_id)
 
-        # oriented on parse_duration()
-        d = re.match(r'P(?P<hours>[0-9]+)H(?P<mins>[0-9]{2})M(?P<secs>[0-9]{2})S',
-                     self._get_property("duration", webpage, fatal=False))
-        if d:
-            hours, mins, secs = d.groups()
-            duration = float(hours) * 60 * 60 + float(mins) * 60 + float(secs)
-        else:
-            duration = None
+        duration = try_get(
+            re.match(r'P(?P<hours>\d+)H(?P<mins>\d+)M(?P<secs>\d+)S',
+                     self._html_search_meta('duration', webpage, fatal=False) or ''),
+            lambda mobj: sum(float(x) * y for x, y in zip(mobj.groups(), (3600, 60, 1))))
+        # _html_search_meta is not used since we don't want name=description to match
+        description = unescapeHTML(
+            self._search_regex(f'meta itemprop="description" content="(.*?)"', webpage, 'description', fatal=False))
 
         return {
             '_type': 'url_transparent',
             'ie_key': DailymotionIE.ie_key(),
             'display_id': video_id,
             'title': self._og_search_title(webpage),
-            'url': self._get_property('embedURL', webpage),
-            'thumbnail': self._get_property('thumbnailURL', webpage, fatal=False),
-            'description': unescapeHTML(self._get_property('description', webpage, fatal=False)),
+            'url': self._html_search_meta('embedURL', webpage),
+            'thumbnail': self._html_search_meta('thumbnailURL', webpage, fatal=False),
```

Comment:
```suggestion
            'thumbnail': self._html_search_meta('thumbnailURL', webpage),
```
that is the default!